src/charm.py

```diff
@@ -47,8 +51,21 @@ def __init__(self, *args) -> None:  # type: ignore[no-untyped-def]
 
         Args:
             args: Variable list of positional arguments passed to the parent constructor.
+
+        Raises:
+            RuntimeError: If initialization of the lightkube client fails
```

Comment:
I think using RuntimeError is too generic, could you create a custom error class (e.g. `LightKubeInitializationError`)

Reply:
Indeed, I changed this to raise a custom error here.